packages/block-editor/src/components/block-list/block-wrapper.js

```diff
@@ -30,7 +30,16 @@ import { BlockContext } from './block';
 import ELEMENTS from './block-elements';
 
 const BlockComponent = forwardRef(
-	( { children, tagName = 'div', __unstableIsHtml, ...props }, wrapper ) => {
+	(
+		{
+			children,
+			tagName = 'div',
+			__unstableIsHtml,
+			wrapperProps: parentWrapperProps,
```

Comment:
Adding this prop is the most controversial change of this PR but the fact is that we need to separate the props applied to the block wrapper (wrapperProps) and the ones applied to the last element rendered by the wrapper (`props` useful to add contentEditable and related props by RichText)

Reply:
Seems like this doesn't work as expected though :(

Reply:
Can we not use the context for this?

Reply:
I'm not sure how you'd do so, retrieve the context value, alter it and add a new provider? It would work but it's also not great code either, I'd rather remove this at some point (after the image refactor).

Reply:
Actually, we can probably use getEditPropsWrapper

Reply:
I meant adding it to `useContext( BlockContext )` below